packages/charts/src/charts/heatmap/heatmap-chart.component.test.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  render as rtlRender,
+  fireEvent,
+  waitFor,
+} from '@testing-library/react';
+
+import { chartData as data } from './heatmap-chart.fixtures';
+
+import HeatmapChart from './heatmap-chart.component';
+
+const render = (overProps: any = {}) => {
+  const labelSelector = 'name';
+  const keys = ['users', 'licenses', 'shops'];
+  const svgDimensions = { width: 700, height: 500 };
+  const theme = {
```

Comment:
Why the theme settings here are not imported from `theme` file ? Also the axes are disabled here - is there any issues with them ?